calendar/engine/availability.go

```diff
@@ -110,7 +110,7 @@ func (m *mscalendar) retrieveUsersToSync(userIndex store.UserIndex, syncJobSumma
 		}
 
 		// If user does not have the proper features enabled, just go to the next one
-		if !(user.Settings.UpdateStatusFromOptions != NotSetStatusOption || user.Settings.ReceiveReminders || user.Settings.SetCustomStatus) {
+		if !(user.IsConfiguredForStatusUpdates() || user.IsConfiguredForCustomStatusUpdates() || user.Settings.ReceiveReminders) {
```

Comment:
I think I would expect these methods to be defined on `user.Settings`, though I don't think it matters. This current way allows for less typing throughout the usages of the method